lib/utils.js

```diff
@@ -187,12 +189,9 @@ function mapOptionFieldNames(options, Model) {
 exports.mapOptionFieldNames = mapOptionFieldNames;
 
 function mapWhereFieldNames(attributes, Model) {
-  let attribute;
-  let rawAttribute;
-
   if (attributes) {
-    for (attribute in attributes) {
-      rawAttribute = Model.rawAttributes[attribute];
+    getComplexKeys(attributes).forEach(attribute => {
```

Comment:
can't we use a `for .. of` loop here? `.foreach` is so 2012

Reply:
No need, JS itself is more than 20 years old :) forEach looks clean here.